docs/libraries/v4-utils-js/compute.md

```diff
@@ -0,0 +1,207 @@
+#  compute
```

Comment:
Idk if we can get this to generate properly, but having this file name (and the other pages) capitalized makes the sidebar look way nicer imo